conferences/types.go

```diff
@@ -47,3 +47,165 @@ type ConferenceSlot struct {
 	// issue sponsor tickets and those cannot be bought individually)
 	AvailableToPublic bool
 }
+
+// ClaimPayment represents a payment for N claims
+type ClaimPayment struct {
+	ID uint64 `gaum:"field_name:id"`
+	// ClaimsPayed would be what in a bill one see as detail.
+	ClaimsPayed []*SlotClaim
```

Comment:
I think this is how this is spelled, or am i miss understanding the meaning of the field 

```suggestion
	ClaimsPaid []*SlotClaim
```